src/yalexs_ble/push.py

```diff
@@ -874,7 +877,12 @@ def update_advertisement(
                 # Encrypted data, we don't know how to decrypt it
                 # but we know its a state change so we schedule an update
                 next_update = HK_UPDATE_COALESCE_SECONDS
-        if YALE_MFR_ID in mfr_data:
+        # Only track the single 0/1 value from the advertisement
+        # as we can get an storm of metadata we don't know how to
+        # decode that starts with b'\x00\x00' and will cause us to
+        # connect over and over again when active scanning is enabled.
+        # b'\x00\x00\x80\x15\xd0\x11\xf7\xa5\x43\x1f\x85\xd7\xff\x23\x5f\x1e\x75\x46'
+        if YALE_MFR_ID in mfr_data and len(mfr_data[YALE_MFR_ID]) == 1:
```

Comment:
@bdraco I'm having trouble starting the yalexs_ble component with this addition, it gets stuck initializing. I have an ASL-05 with always connected enabled, and I'm only seeing advertisements with metadata starting with `0000`. Could this length check be skipped for the first time (i.e. when `self._last_adv_value == -1`)?

Reply:
That should be fine to do. Would you mind doing a PR for that?

Reply:
Sure, I opened https://github.com/bdraco/yalexs-ble/pull/242.

Not sure how far I can get with decoding, I don't think my lock is sending out a timestamp since the values from yesterday and today are identical